pkgs/development/python-modules/testfixtures/default.nix

```diff
@@ -1,23 +1,32 @@
-{ stdenv, buildPythonPackage, fetchPypi
-, mock, manuel, pytest, sybil, zope_component }:
+{ stdenv, buildPythonPackage, fetchPypi, fetchpatch
+, mock, pytest, sybil, zope_component }:
 
 buildPythonPackage rec {
   pname = "testfixtures";
-  version = "6.2.0";
+  version = "6.3.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "7e4df89a8bf8b8905464160f08aff131a36f0b33654fe4f9e4387afe546eae25";
+    sha256 = "1x16xkw483nb1ngv74s7lgaj514pb1ldklal7kb7iwqbxcgnrh2k";
   };
 
-  checkInputs = [ mock manuel pytest sybil zope_component ];
+  checkInputs = [ pytest mock sybil zope_component ];
+
+  patches = [
```

Comment:
Please add a comment with information like:

* why it's needed
* if/when it should be dropped

Reply:
i.e.: fix tests for python3.7 & can removed for the next release.